docs/spark-procedures.md

```diff
@@ -192,6 +192,44 @@ Cherry-pick snapshot 1 with named args
 CALL catalog_name.system.cherrypick_snapshot(snapshot_id => 1, table => 'my_table' )
 ```
 
+### `publish_changes`
+
+Publish changes from a Staged WAP id into the current table state.
+
+publish_changes creates a new snapshot from an existing snapshot without altering or removing the original.
+
+Only append and dynamic overwrite snapshots can be successfully published.
+
+{{< hint info >}}
+This procedure invalidates all cached Spark plans that reference the affected table.
+{{< /hint >}}
+
+#### Usage
+
+| Argument Name | Required? | Type | Description                                     |
```

Comment:
remove extra spaces and keep one trailing space. 

For example,  refer `cherrypick_snapshot`

Reply:
same for all the new tables